src/components/blm/index.jsx

```diff
@@ -25,36 +19,28 @@ class BLMPages extends Component {
             <BLMDirectory />
           </Route>
 
-          <Route path="/blm/education">
-            BLM Education
-          </Route>
+          <Route path="/blm/education">BLM Education</Route>
```

Comment:
Dont change the spacing of the Route components please

Reply:
Got it, just fixed that